priv/templates/partials/head.html.eex

```diff
@@ -3,6 +3,10 @@
   <meta charset="utf-8">
   <meta http-equiv="content-type" content="text/html; charset=utf8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" type="text/css" href="assets/stylesheets/benchee.css">
-  <link rel="stylesheet" type="text/css" href="assets/fontello/css/fontello.css">
+  <%= if inline_assets do %>
+    <%= inline_stylesheets() %>
```

Comment:
I think we could be more explicit here - rather than doing the turn than calling a function etc. we could probably just be like

`File.read "path_as_below"`

For the relevant paths, it's a bit more verbose especially when adding new assets but it's also simpler and more flexible - i.e. if we don't want some JavaScript on some page for instance.

We could also get rid off the repetition of assets between inlining and linking to them (which we don't HAVE to) by declaring a list of assets and then programmatically linking/reading them ion based on the setting.

Reply:
i just wanted to avoid situation when someone adds js or css and forgets to inline it - that's why i'm generating this, but if you want i can make this more static...